common/common.go

```diff
@@ -505,6 +584,37 @@ func containerEngineVersionCheck(containerEngineType string) error {
 	return nil
 }
 
+func initializeContainerBuilder(containerEngineType string) error {
+	// ignoring the error as there could be no such existing builder
+	// valid error would be caught later while creating a builder
+	_ = cleanupContainerBuilder(containerEngineType)
+
+	var args []string
+	args = append(args, "buildx")
+	args = append(args, "create")
+	args = append(args, "--name", BuildxBuilderInstance)
+	args = append(args, "--use")
+
+	_, err := exec.Command(containerEngineType, args...).Output()
```

Comment:
Shouldn't we include driver and the platform parameters here?

FDK Usage for reference ::
docker buildx create --name builderInstance --driver docker-container --platform linux/amd64,linux/arm64

Reply:
Yeah good idea, I'll try that out.